README.md

```diff
@@ -343,6 +346,144 @@ For example, if we named the above XML launch script, `record_all.launch.xml`:
+
+```python
+import launch
+import launch_ros
+import yaml
+
+'''
+Used to load parameters for composable nodes from a standard param file
+'''
+def dump_params(param_file_path, node_name):
+    with open(param_file_path, 'r') as file:
+        return [yaml.safe_load(file)[node_name]['ros__parameters']]
+
+def generate_launch_description():
+    return launch.LaunchDescription([
+        launch.actions.ComposableNodeContainer(
+            name='composable_container',
+            package='rclcpp_components',
+            executable='component_container',
+            composable_node_descriptions=[
+                launch_ros.descriptions.ComposableNode(
+                    package='rosbag2_transport',
+                    plugin='rosbag2_transport::Player',
+                    name='player',
+                    parameters=dump_params("/path/to/params.yaml", "player"),
+                    extra_arguments=[{'use_intra_process_comms': True}]
+                ),
+                # your other components here
+            ]
+        )
+    ])
+}
+```
+
+Here's an example YAML configuration for both composable player and recorder:
+```yaml
+recorder:
+  ros__parameters:
+    use_sim_time: false
+    record:
+      all: true
+      is_discovery_disabled: false
+      rmw_serialization_format: "cdr"
+      topic_polling_interval:
+        sec: 0
+        nsec: 10000000
+      compression_mode: "file"
+      compression_format: "zstd"
+      compression_queue_size: 50
+      compression_threads: 4
+      include_hidden_topics: true
+      include_unpublished_topics: false
+      ignore_leaf_topics: false
+      start_paused: false
+
+    storage:
+      uri: "/path/to/destination/folder"
+      storage_id: "sqlite3"
+      max_cache_size: 20000000
+```
+and
+```yaml
+player:
+  ros__parameters:
+    play:
+      read_ahead_queue_size: 1000
+      node_prefix: ""
+      rate: 1.0
+      loop: false
+      clock_publish_frequency: 1.0
+      clock_publish_on_topic_publish: true
+      delay:
+        sec: 0
+        nsec: 0
+      # Negative timestamps will make the playback to not stop.  
+      playback_duration:
+        sec: -1
+        nsec: 00000000
+      # Negative timestamps will make the playback to not stop.
+      playback_until_timestamp:
+        sec: -1
+        nsec: 00000000
+      start_paused: false
+      start_offset: 
+        sec: 0
+        nsec: 00000000
+      disable_keyboard_controls: true
+      # Negative value means that published messages do not need to be acknowledged.
+      wait_acked_timeout:
+        sec: -1
+        nsec: 00000000
+      disable_loan_message: false
+
+    storage:
+      uri: "path/to/rosbag/file"
+      storage_id: "sqlite3"
+      storage_config_uri: ""
+```
+
```

Comment:
@roncapat Could you please add a few words with the links to the full config files in our tests for reference?

[player_node_params.yaml](https://github.com/ros2/rosbag2/blob/34ccb39463f43a82b6d636ea75661cc6edd6c5f1/rosbag2_transport/test/resources/player_node_params.yaml)

[recorder_node_params.yaml](https://github.com/ros2/rosbag2/blob/34ccb39463f43a82b6d636ea75661cc6edd6c5f1/rosbag2_transport/test/resources/recorder_node_params.yaml)

Reply:
Done.